packages/skin/src/routes/_index/component/card/+page.marko

```diff
@@ -0,0 +1,566 @@
+    </div>
+</span>
+</highlight-code>
+
+    <h2 id="card-self-actionable">Self-Actionable Card</h2>
+
+    <p>
+        The self-actionable card is a variant that allows the entire card to be clickable, leading to a specific action or page. A Self-actionable card <strong>cannot</strong> contain interactive elements such as buttons or links within it.
+    </p>
+    <p>
+        This variant is useful for cards that represent a single item or entity, such as a product or article. To create a self-actionable card, use a <highlight>button</highlight> or <highlight>a</highlight> tag as the container.
+    </p>
+
+<demo>
+    <div style="display: flex; gap: 24px; flex-wrap: wrap;">
+
+    <button class="card">
+        <div class="card__media">
+            <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+        </div>
+        <div class="card__main">
+            <h3 class="card__title">Authentic Rookie Cards</h3>
+            <div class="card__description">
+                Professional-grade rookie cards featuring future Hall of Famers in their awkward rookie photo phase.
+            </div>
+        </div>
+    </button>
+
+    <a class="card" href="https://ebay.com">
+        <div class="card__media">
+            <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+        </div>
+        <div class="card__main">
+            <h3 class="card__title">Authentic Rookie Cards</h3>
+            <div class="card__description">
+                A collection proving that somewhere between a curveball, a three-pointer, and a slap shot lies the secret to happiness.
+            </div>
+        </div>
+    </a>
+
+    </div>
+</demo>
+
+<highlight-code type="html">
+<button class="card">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <h3 class="card__title">Authentic Rookie Cards</h3>
+        <div class="card__description">
+            Professional-grade rookie cards featuring future Hall of Famers in their awkward rookie photo phase.
+        </div>
+    </div>
+</button>
+
+<a class="card" href="https://ebay.com">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <h3 class="card__title">Authentic Rookie Cards</h3>
+        <div class="card__description">
+            A collection proving that somewhere between a curveball, a three-pointer, and a slap shot lies the secret to happiness.
+        </div>
+    </div>
+</a>
+</highlight-code>
+
+    <h2 id="card-hybrid-actionable">Hybrid Actionable Card</h2>
+
+    <p>
+        The hybrid actionable card is a variant that allows the card to be both self-actionable and internally actionable, containing interactive elements such as buttons or links. This is useful for cards that represent a single item or entity, but also need to provide additional actions.
```

Comment:
I think we should clarify that for these hyrbid cards, additional actions are not allowed at all. There can only be one internal action and the card must have the exact same action (added with javascript listener).